.rubocop.yml

```diff
@@ -0,0 +1,1056 @@
+AllCops:
```

Comment:
According to the [Hound website](https://houndci.com/configuration#ruby), this is the rubocop configuration being used.  However, I don't think that's entirely true...  as several cops defined here did not report by Hound.  I have commented out a few cops (commenting them, rather than simply removing them, so that we can see what they are) that we just not working.